src/HKT.ts

```diff
@@ -11,6 +11,10 @@ export interface HKT3<URI, U, L, A> extends HKT2<URI, L, A> {
   readonly _U: U
 }
 
+export interface HKT3<URI, V, U, L, A> extends HKT2<URI, L, A> {
```

Comment:
HKT3 => HKT4 :)

Reply:
I believe in this version, this is correct

https://github.com/gcanti/fp-ts/blob/2fa6998fa2e9b1b2e9bb823afc8888ac5fa5fe39/src/HKT.ts#L14